packages/salesforce/src/Adaptor.js

```diff
@@ -192,20 +243,20 @@ export function bulk(sObjectName, operation, records, options = {}) {
  * `bulkQuery()` uses {@link https://sforce.co/4azgczz Bulk API v.2.0 Query} which is available in API version 47.0 and later.
  * This API is subject to {@link https://sforce.co/4b6kn6z rate limits}.
  * @public
- * @example
- * <caption>The results will be available on `state.data`</caption>
- * bulkQuery(state=> `SELECT Id FROM Patient__c WHERE Health_ID__c = '${state.data.field1}'`);
- * @example
+ * @example <caption>Bulk query patient records where `Health_ID__c` is equal to the value in `state.data.healthId`</caption>
```

Comment:
Update function description and tried using `<code>` instead of `` but it didn't work so i am using `"` as an alternative 
 
<img width="940" alt="Screenshot 2024-12-10 at 1 06 13 PM" src="https://github.com/user-attachments/assets/b27a43c6-1ec9-4747-beb1-2fcfcdb50dd5">